package.json

```diff
@@ -94,11 +94,13 @@
     "boom": "^5.2.0",
     "cids": "^0.5.1",
     "debug": "^3.0.0",
+    "fnv1a": "^1.0.1",
```

Comment:
This is the hash function used by the [go implementation](https://github.com/ipfs/go-ipfs/blob/master/pin/set.go#L31) in serializing the pin set. I just tried to replicate the same process as closely as I could.